package.json

```diff
@@ -92,17 +91,21 @@
         "webpack-dev-server": "^4.5.0"
     },
     "peerDependencies": {
-        "react": "^16.9.0",
-        "react-dom": "^16.9.0"
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0"
     },
     "dependencies": {
+        "@babel/plugin-transform-runtime": "^7.23.6",
+        "@babel/runtime": "^7.23.6",
         "@fortawesome/fontawesome-free": "^5.15.2",
         "@fortawesome/fontawesome-svg-core": "~1.2.34",
         "@fortawesome/free-solid-svg-icons": "^5.15.2",
         "@fortawesome/react-fontawesome": "^0.1.14",
         "@tweakpane/core": "^1.0.8",
         "@tweakpane/plugin-essentials": "^0.1.4",
+        "@types/react-dom": "^18.2.18",
```

Comment:
are types always a devDependency?

Reply:
that's my understanding since typescript isn't runtime

Reply:
then this line can move up out of this section ⬆️

Reply:
OOH yeah, I missed that, thanks